corehq/apps/style/static/style/js/multiselect_utils.js

```diff
@@ -1,6 +1,6 @@
 /* global django:false _:false */
 
-hqDefine('style/js/components/multiselect_utils', function () {
+hqDefine('style/js/multiselect_utils', function () {
```

Comment:
The inline edit widget is in a `components` directory because it uses [Knockout components](http://knockoutjs.com/documentation/component-overview.html), which the js team has talked about using more broadly now that we're firmer on using knockout as our main js library.